### ISSUE TITLE
Incorrect gps coordinates

### ISSUE THREAD

**Issue by rfrelat** [l-gorman/rhomis-R-package#33]
I was comparing the country with the GPS coordinates, and it does not always match. It might be good to check that in pre-processing and set GPS coordinates to NA if the issue can't be easily solved (forgotten '-' in coordinates or longitude and latitude are being mixed-up). 
Here is the current list of households with incorrect gps coordinates (based on country comparing) : [errorCountry.csv](https://github.com/l-gorman/rhomis-R-package/files/9281521/errorCountry.csv).

**Comment by l-gorman** [comment]
Interesting that there is this mismatch. GPS coordinates are not put in manually so it could not be a user error like a forgotten "-". 

I see in this list, the GPS coordinates have been rounded. I think it would good to plot these on a map. I am thinking that the problem could be that these households were initially close to the border, and the rounding put them on the other side of the border. 

I agree though, this type of checking before hand would be really good to implement.

**Comment by rfrelat** [comment]
You are right, most of them are 'close' to the border (especially the burkina faso/ghana border). Yet there are ~50 households that should be in Ethiopia and have coordinates in Niger (14.3°N, 3.97°E). Is it a wrong decimal, with correct coordinates at (14.3°N, 39.7°E)? 

Some households are yet a bit far from the border and it's hard to guess the 'correct' coordinates, e.g. 
the mali's household at 13.77°N, 8.86°E
the malawi household at -1.58°N, 33.72°E
the guatemala household at 14.46 °N, -89.1°E

For those households that are very close to the border (e.g. Burkina Faso household at 11.01°N, -2.09°E), it might be good to manually push them on the 'correct' side of the border (e.g. in that case, 10.98°N, -2.09°E). It would simplify the extraction of GIS information and will be less confusing for other data users. 

Not sure what's the best way to solve this issue in the rhomis package. How do you see it? Let me know if I can be of any help.
